packages/cspell-util-bundle/package.json

```diff
@@ -55,7 +55,7 @@
     "node": ">=12.0.0"
   },
   "devDependencies": {
-    "rimraf": "^2.6.3",
+    "rimraf": "^2.7.1",
```

Comment:
This was just missed. It should be removed or on version `^3.0.2`

Reply:
OK, updated it to 3